core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -775,7 +802,12 @@ private[spark] class MapOutputTrackerMaster(
   override def stop(): Unit = {
     mapOutputRequests.offer(PoisonPill)
     threadpool.shutdown()
-    sendTracker(StopMapOutputTracker)
+    try {
+      sendTracker(StopMapOutputTracker)
+    } catch {
+      case e: Exception =>
```

Comment:
Can we catch a more specific exception? This looks too general.

Reply:
Good catch, I'll narrow this down.

Reply:
So the exception I want to catch is a `SparkException`, so I'll just narrow it to that.

Reply:
Thanks~